src/components/dropdown/dropdown.tsx

```diff
@@ -260,8 +261,12 @@ export class Dropdown implements InteractiveComponent, OpenCloseComponent, Float
   //
   //--------------------------------------------------------------------------
 
-  /** fires when a dropdown item has been selected or deselected */
-  @Event() calciteDropdownSelect: EventEmitter<void>;
+  /**
+   * fires when a dropdown item has been selected or deselected
+   *
```

Comment:
why all the line breaks?

Reply:
vs no line breaks?

![why-not-both-why-not](https://user-images.githubusercontent.com/197440/181344584-d20c3796-4d8e-4463-996e-a3048cc1a78f.gif)

J/k. Force of habit. I find them more readable and easy to add additional notes or tags. I'm sure there's an ESLint rule for this.

Reply:
Oh, I'm also missing some event info here. That's why there's extra, extra newlines. Fixing.

Reply:
Removed extra lines and will rely on the type. Spoke w/ @benelan and @macandcheese about introducing a custom JSDoc tag to provide the interface w/o losing the type reference.